test/iron-location.html

```diff
@@ -93,10 +93,10 @@
     }
 
     /**
-     * We run the iron location tests with a couple different page configurations
-     * (e.g. with and without a base URI), so we define the test set as a function
-     * that we call in each of these configurations.
-     */
+             * We run the iron location tests with a couple different page configurations
```

Comment:
this doesn't look right

Reply:
pls read PR title. formatting is fixed!

I'll take a look at what I flubbed